doc/whats_new.rst

```diff
@@ -217,18 +217,34 @@ Bug
 
 - Fix normalization error in :func:`mne.beamformer.make_lcmv` when ``pick_ori='normal', weight_norm='unit_noise_gain'`` by `Marijn van Vliet`_
 
-- Update MNE-C installation instructions by `buildqa`_
+- Fix MNE-C installation instructions by `buildqa`_
 
 - Fix computation of max-power orientation in :func:`mne.beamformer.make_dics` when ``pick_ori='max-power', weight_norm='unit_noise_gain'`` by `Marijn van Vliet`_
 
 API
 ~~~
 
-- Deprecate ``mne.io.read_annotations_eeglab`` by `Joan Massich`_
+- Deprecated separate reading of annotations and synthesis of STI014 channels in readers by `Joan Massich`_:
```

Comment:
@mmagnuski this is the outline of the changes that would affect `read_raw_eeglab` and related functions. We should figure out if everything will still work for you!

Reply:
Here is a more readable version:

https://10219-1301584-gh.circle-artifacts.com/0/html/whats_new.html#api

Reply:
Thanks, I will try to test on a variety of files during the weekend.